Ginger/GingerAutoPilot/APIModelLib/SwaggerApi/OpenApiBase.cs

```diff
@@ -281,23 +285,58 @@ public ObservableList<OptionalValue> GetListOfParamEnums(SwaggerParameter swagge
                         {
                             Value = item.ToString(),
                             ItemName = item.ToString(),
+                            IsDefault = true
                         };
                         lstOptions.Add(value);
                     }
                 }
-                if (swaggerParameter.Enumeration != null && swaggerParameter.Enumeration.Count > 0)
+                else if (swaggerParameter.Enumeration != null && swaggerParameter.Enumeration.Count > 0)
                 {
                     foreach (object item in swaggerParameter.Enumeration)
                     {
                         OptionalValue value = new OptionalValue()
                         {
                             Value = item.ToString(),
                             ItemName = item.ToString(),
+                            IsDefault = true
```

Comment:
Please check if its setting all values to default as true

Reply:
Setting default only one value.